lib/scarlet/git.rb

```diff
@@ -0,0 +1,14 @@
+class Scarlet
+  module Git
+    def self.get_data
```

Comment:
I'm a bit wary regarding adding more classes for one or two minor uses around the codebase. How about we load these into constants/global config inside the main class? `Scarlet.version` or `Scarlet::SHA/Scarlet::BRANCH`